woocommerce/woocommerce-gutenberg-products-block
# Add Woo icon and branding to block categories

Adds Woo icon branding to the block categories, and styles woo block icons in WooCommerce brand pink/purple (#96588a).

I followed the method in which Jetpack adds it's icon; using `setCategories` instead of PHP since PHP can only add dashicons - the docs recommend using frontend JS to append icons (https://developer.wordpress.org/block-editor/developers/filters/block-filters/).


Closes #569



#### Accessibility

WordPress adds aria-hidden to the icon component.

### Screenshots

![Edit Post ‹ local wordpress test — WordPress 2019-05-21 10-44-00](https://user-images.githubusercontent.com/90977/58086280-40075f00-7bb6-11e9-8587-e6cc49347353.png)

### How to test the changes in this Pull Request:

1. Go to the editor
2. Click the + to view blocks
3. Scroll to the WooCommerce section 
4. Confirm icon is shown. It should have height 20px.
5. Open WooCommerce section. Confirm icons are purple.
